fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
Very well. I filed #213 (and an associated Pagure issue) to track that. I still don't think the current API here makes sense, though.

I'm guessing there's a time crunch here. When does this need to be in production?